regulatory/data/history.yml

```diff
@@ -0,0 +1,35 @@
+change_requests:
```

Comment:
Could we add a comment to the top of this file explaining what it's about?  And/or, add a README.md to the `regulatory/` directory with a pointer to documentation for the tool(s) we're using?

Reply:
(Same elsewhere)